Sources/Brave/Frontend/Browser/Helpers/LaunchHelper.swift

```diff
@@ -127,6 +123,27 @@ public actor LaunchHelper {
   }
 }
 
+private extension FilterListStorage {
+  /// Return all the blocklist types that are valid for filter lists.
+  var validBlocklistTypes: Set<ContentBlockerManager.BlocklistType> {
+    if filterLists.isEmpty {
+      /// If we don't have filter lists yet loaded, use the settings
```

Comment:
nit: two slashes for standard comments, three are only used for doc

Reply:
Oops ya sorry